README.md

```diff
@@ -19,16 +19,18 @@ The `/search` API endpoint has few additional query parameters. More might be ad
   By default this endpoint always returns only the newest compatible package.
 * `category`: Filters the package by the given category. Available categories can be seend when going to `/categories` endpoint.
 * `package`: Filters by a specific package name, for example `mysql`. Returns the most recent version.
-* `all`: This can be set to true to list all package versions. This is set to `false` by default.
-* `experimental`: This can be set to true to list packages considered to be experimental. This is set to `false` by default.
+* `all`: This can be set to `true` to list all package versions. This is set to `false` by default.
+* `prerelease`: This can be set to `true` to list prerelease versions of packages. This is set to `false` by default.
```

Comment:
Could you define what "prerelease versions" means. Like say that it mean versions <1.0 and packages using the deprecated `beta` and `experimental` release tags (IIRC).

Reply:
Added description. The registry fully relies on semantic versioning to decide if the version of a package is a prerelease, release tags are not considered. This is why it is important to have consistency between release tags and versions till we fully sunset versions that rely on release tags.